vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -280,6 +281,28 @@ void TextInputShadowNode::registerEvents() {
         }
       });
 
+  m_controlKeyDownRevoker = control.KeyDown(
+      winrt::auto_revoke, [=](auto &&, winrt::KeyRoutedEventArgs const &args) {
+        if (args.Key() == winrt::VirtualKey::Enter && !args.Handled()) {
+          auto instance = wkinstance.lock();
+          folly::dynamic eventDataSubmitEditing = {};
+          if (m_isTextBox) {
+            eventDataSubmitEditing = folly::dynamic::object("target", tag)(
+                "text", HstringToDynamic(control.as<winrt::TextBox>().Text()));
+          } else {
+            eventDataSubmitEditing = folly::dynamic::object("target", tag)(
+                "text",
+                HstringToDynamic(control.as<winrt::PasswordBox>().Password()));
+          }
+          if (!m_updating && instance != nullptr) {
+            instance->DispatchEvent(
```

Comment:
Don't raise the event if multline flag is specified

Reply:
This is covered in checking if the keydown is already handled above. True for Multiline since enter key is used for text editing.

---
In reply to: [335120678](https://github.com/microsoft/react-native-windows/pull/3423#discussion_r335120678) [](ancestors = 335120678)